test/views/components/DownloadLinkSpec.scala

```diff
@@ -17,38 +17,58 @@
 package views.components
 
 import helpers.Formatters
-import models.FileFormat.Pdf
-import models.{C79Certificate, FileFormat, VatCertificateFile, VatCertificateFileMetadata}
+import models.FileFormat.{Csv, Pdf}
+import models._
```

Comment:
Its better to revert back to only importing specific models that we need, as its good practice to provide the specific models (that are being used) and also it increases the compile time.